src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/SpecificPriceController.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Controller\Admin\Sell\Catalog\Product;
+
+use DateTimeInterface;
+use Exception;
+use PrestaShop\Decimal\DecimalNumber;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\Exception\SpecificPriceConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\Query\GetSpecificPriceList;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\QueryResult\SpecificPriceList;
+use PrestaShop\PrestaShop\Core\Domain\ValueObject\Reduction;
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\FormBuilderInterface;
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\FormHandlerInterface;
+use PrestaShop\PrestaShop\Core\Util\DateTime\DateTime as DateTimeUtil;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class SpecificPriceController extends FrameworkBundleAdminController
+{
+    public function listAction(int $productId): JsonResponse
+    {
+        $specificPricesList = $this->getQueryBus()->handle(
+            new GetSpecificPriceList($productId, $this->getContextLangId())
+        );
+
+        return $this->json(['specificPrices' => $this->formatSpecificPricesList($specificPricesList)]);
+    }
+
+    public function createAction(Request $request, int $productId): Response
+    {
+        $form = $this->getFormBuilder()->getForm(['product_id' => $productId]);
+        $form->handleRequest($request);
+
+        try {
+            $result = $this->getFormHandler()->handle($form);
+
+            if ($result->isSubmitted() && $result->isValid()) {
+                $this->addFlash('success', $this->trans('Creation successful', 'Admin.Notifications.Success'));
```

Comment:
```suggestion
                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
```
This string (the other way around) already exists in Admin.Notifications.Success. Let's use it to avoid duplicating it. ☺️